ichnaea/geoip.py

```diff
@@ -164,7 +164,7 @@
 }
 
 
-def configure_geoip(filename=GEOIP_PATH, mode=MODE_AUTO,
+def configure_geoip(filename=None, mode=MODE_AUTO,
```

Comment:
`GEOIP_PATH` is a setting and we don't want the value at the time this module is loaded to persist as an argument default.